website_crm_privacy_policy/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Jorge Camacho <jcamacho@trey.es>
+# Copyright 2015 Antonio Espinosa <antonioea@antiun.com>
+# Copyright 2016-2018 Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Website CRM privacy policy",
+    "category": "Website",
+    "version": "11.0.1.0.0",
+    "website": "http://www.trey.es",
```

Comment:
This should be https://odoo-community.org/

Reply:
It's better to put the repository, but you can leave it this time: https://github.com/OCA/website.